CONTRIBUTING.md

```diff
@@ -64,20 +76,55 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Member and Collaborator
+Collaborators are expected to follow this policy and continue to send pull requests, go through proper review, and have other collaborators merge their pull requests.
+
+### Becoming a Contributor
+
+To begin contributing to working groups and initiatives, start participating!
+
+Attend the meetings of the working group or initiative, investigate issues tagged
+as `good first issue`, open issues and pull requests, or provide insight via GitHub.
+
+Individuals can open an issue asking to be an Observer of CommComm meetings. You
+can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142)!
+
+### Becoming a Collaborator
+
+CommComm Collaborators are active contributors with write access to the repo(s)
+relevant to their initiative or working group.
+
+Active Contributors of a CommComm initiative or working group can be nominated by
+another Collaborator to become a Collaborator of that initiative or working group.
+On approval by the consensus process, the Contributor becomes a Collaborator.
+
+Typical activities of a Collaborator include:
+
+* Helping users and novice contributors
+* Contributing code, non-code, and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Participating in working groups or initiatives
+* Merging pull requests
```

Comment:
Not sure if we should add more context around this, like once consensus has been achieved 🤔